src/upgrades/pre_0_14_0/old_commands.rs

```diff
@@ -133,7 +133,7 @@ pub enum Pre0_14_0CertAuthStorableCommand {
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct Pre0_14_0AspaDefinitionUpdates {
     add_or_replace: Vec<Pre0_14_0AspaDefinition>,
-    remove: Vec<Pre0_14_0ProviderAs>, // was using string notation
+    remove: Vec<rpki::resources::Asn>,
```

Comment:
What is this change?

Reply:
It was a bug. Krill > 0.10.0 use a u32 here and not an ASN string, as 0.9.x did

Reply:
our regression test code did not include any "delete ASPA" (by "remove") in its history.